instances/experimentalApproach/electrophysiology.jsonld

```diff
@@ -6,9 +6,9 @@
   "@type": "https://openminds.ebrains.eu/controlledTerms/ExperimentalApproach",
   "definition": "Any experimental approach focused on electrical phenomena associated with living systems, most notably the nervous system, cardiac system, and musculoskeletal system.",
   "description": null,
-  "interlexIdentifier": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0741202",
   "knowledgeSpaceLink": null,
   "name": "electrophysiology",
-  "preferredOntologyIdentifier": "http://uri.interlex.org/base/ilx_0741202",
+  "preferredOntologyIdentifier": "	http://uri.interlex.org/tgbugs/uris/readable/modality/Electrophysiology",
```

Comment:
I think there are too many spaces here

Reply:
Yes, happened while copy-pasting. Fixed it.